server/app/services/question/types/NullQuestionDefinitionConfig.java

```diff
@@ -0,0 +1,145 @@
+package services.question.types;
+
+import java.time.Instant;
+import java.util.Locale;
+import java.util.Optional;
+import java.util.OptionalLong;
+import services.LocalizedStrings;
+
+/**
+ * This is an empty QuestionDefinitionConfig that is used when the system can't find the question.
+ * It works with the NullQuestionDefinition class to provide a placeholder object that assists in
+ * letting the application to continue loading in the event of an error.
+ *
+ * <p>In normal operation this shouldn't ever be reached, but an occasional gremlin has resulted in
+ * the rare instance of a program pointing at an old version of a question.
+ */
+public class NullQuestionDefinitionConfig extends QuestionDefinitionConfig {
+  private final long _id;
```

Comment:
**nitpick:** underscored-prefixed variable names are not an idiom we use in the java codebase https://google.github.io/styleguide/javaguide.html#s5.1-identifier-names
```suggestion
  private final long id;
```